src/test/java/homework17.java

```diff
@@ -0,0 +1,51 @@
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.openqa.selenium.By;
+import org.testng.annotations.Test;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import java.time.Duration;
+import org.openqa.selenium.chromium.ChromiumDriver;
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+public class homework17 extends BaseTest {
+    @Test
+public static void addSongToPlaylist() throws InterruptedException {
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)